cross/device-mapper/PLIST

```diff
@@ -0,0 +1,5 @@
+lnk:lib/libdevmapper.so
+lib:lib/libdevmapper.so.1.02
+bin:sbin/blkdeactivate
+bin:sbin/dmsetup
+bin:sbin/dmstats
```

Comment:
Are the binary needed or only the libraries are?

Reply:
I don't see these being referenced anywhere so I've removed them